File: Smaller and Larger/Smaller and Larger.py
def getMoreAndLess(arr, x):
    # code here
    less = 0
    greater = 0
    for i in arr:
        if i >= x:
            greater +=1
        if i <=x:
            less +=1
    return less,greater

array = (1,2,3,5,6,7)

print (getMoreAndLess(array,2))

File: Smaller and Larger/Smaller and Larger.cpp
#include <iostream>

using namespace std;

int main() {
    int x = 3 ; 
    int greater = 0;
    int less = 0 ;
    int arr[] = {23,152,12412,123};
    int n = 4;

    for (int i=0;i<n;i++){
        if (arr[i] >= x){
            greater+=1;
        }
        if(arr[i] <= x){
            less +=1 ; 
            
    }
    }
    cout<<less<<" "<<greater;
    
    }
